src/test/gov/usgs/wma/waterdata/DBTestConfig.java

```diff
@@ -0,0 +1,45 @@
+package gov.usgs.wma.waterdata;
```

Comment:
It looks like the test java classes are not under /src/test/java/...

Reply:
Ah, good catch.  Will update this.